test/controllers/user_role_controller_test.exs

```diff
@@ -1,57 +1,41 @@
 defmodule CodeCorps.UserRoleControllerTest do
-  use CodeCorps.ApiCase
+  use CodeCorps.ApiCase, resource_name: :user_role
 
-  alias CodeCorps.UserRole
-  alias CodeCorps.Repo
-
-  defp build_payload, do: %{ "data" => %{"type" => "user-role", "attributes" => %{}}}
+  @invalid_attrs %{}
 
   describe "index" do
     test "lists all entries on index", %{conn: conn} do
       path = conn |> user_role_path(:index)
-      json = conn |> get(path) |> json_response(200)
+      conn = conn |> get(path)
 
-      assert json["data"] == []
+      assert json_response(conn, 200)["data"] == []
     end
 
     test "filters resources on index", %{conn: conn} do
-      designer = insert(:role, name: "Designer")
-      lawyer = insert(:role, name: "Lawyer")
-      photographer = insert(:role, name: "Photographer")
-
-      user = insert(:user)
-      user_role_1 = insert(:user_role, user: user, role: designer)
-      user_role_2 = insert(:user_role, user: user, role: lawyer)
-      insert(:user_role, user: user, role: photographer)
+      [user_role_1, user_role_2] = insert_pair(:user_role)
+      insert(:user_role)
```

Comment:
Again, a link to https://github.com/code-corps/code-corps-api/pull/425/files#r86388037 as a pointer on how to write this in one line.